kubetest/main.go

```diff
@@ -168,6 +172,7 @@ func defineFlags() *options {
 	flag.StringVar(&o.nodeTestArgs, "node-test-args", "", "Test args specifically for node e2e tests.")
 	flag.BoolVar(&o.noAllowDup, "no-allow-dup", false, "if set --allow-dup will not be passed to push-build and --stage will error if the build already exists on the gcs path")
 	flag.BoolVar(&o.nodeTests, "node-tests", false, "If true, run node-e2e tests.")
+	flag.StringVar(&o.numNodes, "num-nodes", "4", "Number of nodes to create.")
```

Comment:
can we make it clear what this flag is for? kubetest has too many flags :|

Reply:
Maybe it would be better to eliminate this flag to keep things simple? This num-nodes option sets the number of worker nodes to spin up in the cluster. For our network connectivity testing, 2 worker nodes is sufficient. The only reason to keep it would be if other, future multi-node test cases need to have this configurable.